tests/functional/tests.d/340-selfaccesses.sh

```diff
@@ -161,7 +161,7 @@ testsuite_selfaccesses()
     set -e
 
     success ssh shellaccountatlo2_mustwork   $a0 $shellaccount@127.0.0.2 --kbd-interactive -- echo $randomstr
-    contain REGEX "$shellaccount@($targethostname|127.0.0.2):22"
+    contain REGEX "$shellaccount@($targethostname|127.0.0.2|fv-[a-z0-9]+):22"
```

Comment:
```suggestion
    contain REGEX "$shellaccount@($targethostname|127.0.0.2|fv-[a-z0-9-]+):22"
```

It may contains a dash as well: ```fv-az120-539```